site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Installing Agones on Azure Kubernetes Service using Terraform"
+linkTitle: "Azure"
+weight: 20
+description: >
+  You can deploy Kubernetes cluster on Azure Kubernetes Service and install Agones using Terraform.
```

Comment:
The GKE description is a bit more concise: "You can use Terraform to provision a GKE cluster and install Agones on it." Can you tweak this one to match?